test/integration/targets/nxos_vlans/tests/cli/overridden.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- debug:
+    msg: "Start nxos_vlans overridden integration tests connection={{ ansible_connection }}"
+
+- name: setup1
+  cli_config: &cleanup
+    config: |
+      no vlan 5
+      no vlan 6
+      no vlan 9
+
+- block:
+  - name: setup
+    cli_config:
+      config: |
+        vlan 5
+        name test-vlan5
+        state suspend
+        vlan 6
+
+  - name: Gather vlans facts
+    nxos_facts: &facts
+      gather_subset:
+        - '!all'
+        - '!min'
+      gather_network_resources: vlans
+
+  - name: Overridden
+    nxos_vlans: &overridden
+      config:
+        - vlan_id: 9
+          name: test-vlan9
+          enabled: false
+      state: overridden
+    register: result
+
+  - assert:
```

Comment:
We also need to assert that no extra command is sent.

Reply:
Fixed